databricks/sdk/_base_client.py

```diff
@@ -130,6 +131,14 @@ def flatten_dict(d: Dict[str, Any]) -> Dict[str, Any]:
         flattened = dict(flatten_dict(with_fixed_bools))
         return flattened
 
+    @staticmethod
+    def _is_seekable_stream(data) -> bool:
```

Comment:
We might also want to add a test for this method, passing common types: string, byte array, buffered reader etc.

Reply:
Done.